R/sjImportSPSS.R

```diff
@@ -601,7 +604,10 @@ sji.getValueLabel <- function(x, attr.only = TRUE, include.values = NULL) {
         }
         # here we include values as prefix of labels
         if (include.values == "as.prefix" || include.values == "p") {
-          labels <- sprintf("[%i] %s", values[reihenfolge], labels)
+          if (is.numeric(x))

```

Comment:
I guess, here it should be `if (is.numeric(values))` to be safe, because if `x` is a factor with numeric values, it won't be recognized here.